hugegraph-dist/src/assembly/static/scripts/example.groovy

```diff
@@ -15,53 +15,55 @@
  * under the License.
  */
 import org.apache.hugegraph.HugeFactory
+import org.apache.hugegraph.backend.id.IdGenerator
 import org.apache.hugegraph.dist.RegisterUtil
+import org.apache.hugegraph.type.define.NodeRole
 import org.apache.tinkerpop.gremlin.structure.T
 
-RegisterUtil.registerCassandra();
-RegisterUtil.registerScyllaDB();
+RegisterUtil.registerRocksDB()
 
 conf = "conf/hugegraph.properties"
-graph = HugeFactory.open(conf);
-schema = graph.schema();
+graph = HugeFactory.open(conf)
+graph.serverStarted(IdGenerator.of("server-tinkerpop"), NodeRole.MASTER)
```

Comment:
@zyxxoo we must start server with the node-role now？

Reply:
verification mean config
auth.authenticator= org.apache.hugegraph.auth.StandardAuthenticator